src/lamp_py/performance_manager/alerts.py

```diff
@@ -335,9 +335,9 @@ def extract_start_end(
     for key in timestamp_columns:
         timestamp_key = f"{key}_timestamp"
         alerts[key] = (
-            pandas.to_datetime(alerts[timestamp_key], unit="s", utc=True)
+            pandas.to_datetime(alerts[timestamp_key], unit="s")
+            .dt.tz_localize("UTC", ambiguous="infer")
             .dt.tz_convert("America/New_York")
-            .dt.floor("s")
```

Comment:
the way this was written was causing problems around DST. now use `tz_localize` to set the timezone to utc and infer DST handling. then convert to east coast time